### PR TITLE
Update guild.tsx

### PR DESCRIPTION
**Pull request by BOBOtoTHEmoon**

### CODE REVIEW

**Review by MantisClone — CHANGES_REQUESTED** [review]
@BOBOtoTHEmoon Please just remove the section instead of just commenting it out.

Also, this PR fails to build.  Please make sure the PRs build and deploy successfully.

File: components/about-us/sub-pages/guilds/guild.tsx [hunk 1]
```diff
@@ -133,6 +125,14 @@ function Guild() {
...
             alt='writers-guild'
           />
+          <Section
+            title='Analytics Guild (INACTIVE)'
+            text='The Analytics Guild builds data products and uses data analytics to equip
+                    bDAO members to achieve our mission with data-driven intelligence.
+                    The Analytics Guild also incubated DAO DASH.'
+            img='/images/analytic-guild.png'
+            alt='analytics-guild'
+          />
```
Comment (MantisClone) [inline components/about-us/sub-pages/guilds/guild.tsx:135]:
```suggestion
            reverse={true}
          />
```

**Review by MantisClone — CHANGES_REQUESTED** [review]

**Review by MantisClone — APPROVED** [review]

**Comment by MantisClone** [comment]
@BOBOtoTHEmoon I reversed the Analytics Guild so that it matched the pattern.

**Comment by BOBOtoTHEmoon** [comment]
@MantisClone okay sir 👍🏼

**Pull request merged**